connect/runtime/src/test/java/org/apache/kafka/connect/runtime/SourceTaskOffsetCommitterTest.java

```diff
@@ -82,21 +84,20 @@ public void setup() {
     }
 
     @Test
-    public void testSchedule() throws Exception {
+    public void testSchedule() {
         Capture<Runnable> taskWrapper = EasyMock.newCapture();
 
         EasyMock.expect(executor.scheduleWithFixedDelay(
                 EasyMock.capture(taskWrapper), eq(DEFAULT_OFFSET_COMMIT_INTERVAL_MS),
                 eq(DEFAULT_OFFSET_COMMIT_INTERVAL_MS), eq(TimeUnit.MILLISECONDS))
         ).andReturn(commitFuture);
 
-        EasyMock.expect(committers.put(taskId, commitFuture)).andReturn(null);
-
         PowerMock.replayAll();
 
         committer.schedule(taskId, task);
         assertTrue(taskWrapper.hasCaptured());
         assertNotNull(taskWrapper.getValue());
+        assertEquals(Utils.mkMap(mkEntry(taskId, commitFuture)), committers);
```

Comment:
mkMap and mkEntry are both statics in `Utils`, why do we have static import for one and not the other? Anyway, perhaps this could just be `Collections.singletonMap`?

Reply:
Good point, changed to use `singletonMap`. Also removed some imports and an unnecessary @PrepareForTest.